.travis.yml

```diff
@@ -41,7 +42,7 @@ matrix:
       env: TARGET=docs
     - php: '7.3'
       env: TARGET=lint
-    - php: '7.2'
+    - php: '7.1'
```

Comment:
Hmm 🤔

Reply:
Oh that's weird… what happened?

Reply:
`3.x` wasn't managed by dev kit before, right?

Reply:
Ah yeah that's it!